Tasks/CondaAuthenticateV0/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -3,6 +3,11 @@
   "loc.helpMarkDown": "",
   "loc.description": "Authentication task for the conda client",
   "loc.instanceNameFormat": "Conda Authenticate",
+  "loc.input.label.workloadIdentityServiceConnection": "'Entra Workload ID-backed Azure DevOps user' Service Connection",
+  "loc.input.help.workloadIdentityServiceConnection": "If this is set, it will generate credentials using service connection tenant ID.",
```

Comment:
tenant id ? I think this should say something like:
``it will generate credentials using the service connection.``
``it will generate credentials using the service connection instead of the build pipeline identity.``
``it will generate credentials using the service connection instead of the System.AccessToken.``

Reply:
Yes I was going to ask you about the correct wording.